pkg/controller/pod.go

```diff
@@ -23,6 +24,12 @@ const (
 	annotationAppMeshVirtualNodeName = "appmesh.k8s.aws/virtualNode"
 )
 
+
+type CloudMapInstanceCacheItem struct {
```

Comment:
why they have different cases.. like "key" and "InstanceSummary", but i didn't see there are treated differently in code. (note, in golang field/structs starts with upper case means "public" in other langs)
if CloudMapInstanceCacheItem is not supposed to be used outside of this package, it's better to be `cloudMapInstanceCacheItem`

Reply:
Was in initially in a different file before I moved it here. So, the case carried over, will correct it.

Reply:
humm, it's still CloudMapInstanceCacheItem(upper case :D)

Reply:
Yeah it is used in controller.go